x-pack/plugins/security_solution/public/network/pages/network.test.tsx

```diff
@@ -26,7 +26,7 @@ import { Network } from './network';
 import { NetworkRoutes } from './navigation';
 import { mockCasesContract } from '../../../../cases/public/mocks';
 import { APP_UI_ID, SecurityPageName } from '../../../common/constants';
-import { getAppLandingUrl } from '../../common/components/link_to/redirect_to_overview';
+import { getAppLandingUrl } from '../../common/components/link_to';
```

Comment:
shouldn't the import be `from '../../common/components/link_to/redirect_to_landing';`